src/connection/WalletConnect.ts

```diff
@@ -31,7 +31,7 @@ export class WalletConnectPopup extends WalletConnect {
 }
 
 // Custom class for Uniswap Wallet specific functionality
-export class UniwalletConnect extends WalletConnectPopup {
+export class UniwalletConnect extends WalletConnectV1WithAnalytics {
```

Comment:
Why isn't this being migrated to v2? Should a TODO/ticket be added/created?

Reply:
https://linear.app/uniswap/issue/WEB-2346/update-uniswap-wallet-to-walletconnect-v2

https://github.com/Uniswap/interface/pull/6762